rust/connlib/shared/Cargo.toml

```diff
@@ -28,7 +29,7 @@ resolv-conf = "0.7.0"
 serde = { version = "1.0", default-features = false, features = ["derive", "std"] }
 serde_json = { version = "1.0", default-features = false, features = ["std"] }
 thiserror = { version = "1.0", default-features = false }
-tokio = { version = "1.36", default-features = false, features = ["rt", "rt-multi-thread"]}
+tokio = { version = "1.36", default-features = false, features = ["fs", "rt", "rt-multi-thread"]}
```

Comment:
This isn't part of the change but I noticed it when running Cargo clippy, I guess the dependency solver had been adding `fs` to Tokio but this crate wasn't asking for it directly.

Is there a way to find these problems automatically?